release/models/ospf/openconfig-ospfv2-area-interface.yang

```diff
@@ -227,6 +233,58 @@ submodule openconfig-ospfv2-area-interface {
     }
   }
 
+  grouping ospfv2-area-interface-authentication-config {
+    description
+      "Configuration parameters relating to OSPF interface
+      authentication";
+
+    leaf auth-mode {
+      type identityref {
+        base oc-ospf-types:AUTH_MODE;
+      }
+      description
+        "Authentication mode to be used for OSPF interface links.";
+    }
+
+    leaf simple-password {
+      type string;
+      description
+        "Password for simple authentication mode to be specified for OSPF
+        authentication.";
+    }
+  }
+
+  grouping ospfv2-area-interface-message-digest-config {
+    description
+      "Configuration parameters relating to OSPF message-digest
+      authentication";
+
+    leaf key-id {
+      type uint8 {
+        range "1..255";
```

Comment:
This range looks to be EOS specific.  The entire 8-bit range is usable including 0 for the Key ID

Reply:
Thanks, I have removed the restriction.